java/src/server/facade/ServerFacade.java

```diff
@@ -141,8 +142,19 @@ public void join(int gameID, CatanColor color) throws JoinGameException {
      * @throws SendChatException
      */
     @Override
-    public void sendChat(int gameID, int player, String message) throws SendChatException {
+    public GameModelDTO sendChat(int gameID, int player, String message) throws SendChatException {
+        Game game = gameManager.getGameByID(gameID);
 
+        try {
+            String playerName = game.getPlayerNameByIndex(player);
+            MessageLine line = new MessageLine(playerName, message);
+            game.getChat().addMessage(line);
+        } catch (PlayerExistsException e) {
+            e.printStackTrace();
+            throw new SendChatException("Failed to send the chat message!");
+        }
+
+        return game.getDTO();
```

Comment:
this should be in the try, that way when an exception is thrown it doesn't still send back the json